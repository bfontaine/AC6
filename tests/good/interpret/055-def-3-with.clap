def  f (x) (y) = g(x) + h(y)
with g (x) = x + h(x)
with h (x) = 42 - x

val q = f (1) (42)

