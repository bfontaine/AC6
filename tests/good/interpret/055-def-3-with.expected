:- f = <code>
:- g = <code>
:- h = <code>
:- q = 42
